boards/arm/thingy53_nrf5340/board.c

```diff
@@ -95,20 +75,6 @@ static int setup(const struct device *dev)
 	const struct device *gpio;
 	int err;
 
-	gpio = DEVICE_DT_GET(NRF21540FEM_CTRL_PDN_GPIO_NODE);
-	if (!device_is_ready(gpio)) {
-		LOG_ERR("%s device not ready", gpio->name);
-		return -ENODEV;
-	}
-	err = gpio_pin_configure(gpio, NRF21540FEM_CTRL_PDN_PIN,
-				 NRF21540FEM_CTRL_PDN_FLAGS |
-				 IS_ENABLED(CONFIG_THINGY53_MISO_WORKAROUND) ?
```

Comment:
Could you please remove `CONFIG_THINGY53_MISO_WORKAROUND` form thingy's Kconfig?